app/src/main/res/values/strings.xml

```diff
@@ -104,4 +104,6 @@
     <string name="msg_close_drawer">close</string>
     <string name="title_pending">Pending</string>
     <string name="title_completed">Completed</string>
+    <string name="cancel">Cancel</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.cancel` appears to be unused
 Unused resources make applications larger and slow down builds.